### PR TITLE
Fixed missing UI URL in localcluster-deployment.md

### LINKED ISSUE #2209: Deploying Kubeflow Pipelines Standalone on a local cluster with kind, k3s, and k3s on WSL
https://www.kubeflow.org/docs/pipelines/installation/localcluster-deployment/

At the very end after deploying kubeflow, it doesn't list the URL with which the UI can be accessed

### PR DESCRIPTION
**Pull request by alfsuse**
Fixed missing URL to access UI at the end of the documentation as per https://github.com/kubeflow/website/issues/2209  and added an alternative method as a note to access K3s from WSL2.

Fixes #2209

### CODE REVIEW

**Comment by 8bitmp3** [comment]
Thanks @alfsuse. I'm going through it and will send a PR, having tested the file with a Markdown linter. 

To run kubectl in Windows Terminal, can you still test the installation with `kubectl version --client`?

File: content/en/docs/pipelines/installation/localcluster-deployment.md [hunk 1]
```diff
@@ -266,7 +266,14 @@ To set up access to your WSL instance:
...
 
-3. Run kubectl in a Windows terminal. If you don't kubectl 
+**Note:**:An alternative method to the above points is to execute the following in a Powershell Terminal
+     
+     ```SHELL
+     $env:KUBECONFIG='//wsl$/saio-wsl/etc/rancher/k3s/k3s.yaml'
+     sc //wsl$/saio-wsl/etc/rancher/k3s/k3s.yaml ((gc -raw //wsl$/saio-wsl/etc/rancher/k3s/k3s.yaml) -replace '127.0.0.1','localhost')
+     ```
+     
+3. Run kubectl in a Windows terminal. If you don't have kubectl
```
Comment (8bitmp3) [inline content/en/docs/pipelines/installation/localcluster-deployment.md:276]:
```suggestion
    **Note:** Alternatively, you can run the following commands in the PowerShell
console:
  
    ```powershell
    $env:KUBECONFIG='//wsl$/saio-wsl/etc/rancher/k3s/k3s.yaml'
    sc //wsl$/saio-wsl/etc/rancher/k3s/k3s.yaml ((gc -raw //wsl$/saio-wsl/etc/rancher/k3s/k3s.yaml) -replace '127.0.0.1','localhost')
    ```

3. Run kubectl in Windows Terminal. If you don't have kubectl 
```

I'm going to send your a PR with small changes to the rest of the file, so this is just an fyi

File: content/en/docs/pipelines/installation/localcluster-deployment.md [hunk 2]
```diff
@@ -291,6 +298,12 @@ arise using it.
     ```
 
     The Kubeflow Pipelines deployment may take several minutes to complete.
+2. Verify that the Kubeflow Pipelines UI is accessible by port-forwarding:
+
+   ```SHELL
+   kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80
+   ```
+Open the Kubeflow Pipelines UI at http://localhost:8080/ or if you are using Kind or K3s within a VM http://{VM IP ADDRESS}:8080/
```
Comment (8bitmp3) [inline content/en/docs/pipelines/installation/localcluster-deployment.md:306]: fyi

```suggestion
2. Verify that the Kubeflow Pipelines UI is accessible by port-forwarding:

    ```SHELL
    kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80
    ```

    Then, open the Kubeflow Pipelines UI at `http://localhost:8080/`. If you are
    using kind or K3s in a virtual machine (VM), you can access the web UI at
    `http://{VM_IP_ADDRESS}:8080/` (replace `VM_IP_ADDRESS` with your VM's IP
    address.)

    **Note**: `kubectl apply -k` accepts local paths and paths that are
    formatted as [hashicorp/go-getter
    URLs](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md#url-format).
    While the paths in the preceding commands look like URLs, they are not
    valid URLs.
```

I'm going to send your a PR with small changes to the rest of the file, so this is just an fyi

**Review by 8bitmp3 — COMMENTED** [review]
I'm going to send you a PR with small changes to the rest of the file, so this is just an fyi

@alfsuse Update: please take a look - preliminary work before I do a pull request with your branch https://github.com/8bitmp3/website/commit/6f20ab38e8bac5092e07673a71729cdda42c0ca0

**Comment by alfsuse** [comment]
@8bitmp3 this looks great to me

> Thanks @alfsuse. I'm going through it and will send a PR, having tested the file with a Markdown linter.

  cool! wait for it

> To run kubectl in Windows Terminal, can you still test the installation with `kubectl version --client`?
Yep just checked.

**Comment by 8bitmp3** [comment]
thank you @alfsuse !

cc @joeliedtke  @Bobgy @IronPan @neuromage @Ark-kun @numerology please take a look, thank you!
(OWNERS from https://github.com/kubeflow/website/blob/master/content/en/docs/pipelines/OWNERS)

/ok-to-test

**Comment by Bobgy** [comment]
Thank you for the contribution!
Would you mind resolving the conflicts?

**Comment by alfsuse** [comment]
@Bobgy   @8bitmp3  fixed as per suggestion.
I would say that /lgtm to me

**Comment by Bobgy** [comment]
/approve
/Lgtm

Thank you!

**Pull request merged**